### PR TITLE
Created a meeting form

### PR DESCRIPTION
**Pull request by JorPoon**
# Description

The goal was to create a meetings form.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change status
- [x] Complete, tested, ready to review and merge but subject to change from feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Tested out on localhost, form is rendering and can input text into separate fields

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/components/MeetingsComponents/MeetingsForm.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        title: '',
+        startTime: '',
+        endTime: '',
+        location: '',
+        description: '',
+        
+    }
+
+    render() {
+        return(
```
Comment (Estrax) [inline src/components/MeetingsComponents/MeetingsForm.js]: make sure you use consistent spacing and proper indentations for all the elements in the app.

**Review by Estrax — DISMISSED** [review]

File: src/components/MeetingsComponents/MeetingsForm.js (hunk 1 above)
Comment (Estrax) [reply src/components/MeetingsComponents/MeetingsForm.js]: ... or at least set up a proper linter

Comment (JorPoon) [reply src/components/MeetingsComponents/MeetingsForm.js]: what are the linter extensions you use?

Comment (JorPoon) [reply src/components/MeetingsComponents/MeetingsForm.js]: Ok I got it, my formatter was not working

**Review by Estrax — APPROVED** [review]

**Review by tfolbrecht — APPROVED** [review]
Nice

**Pull request merged**